manifests/profile/haproxy.pp

```diff
@@ -52,4 +52,26 @@
     notify  => Service['haproxy'],
   }
 
+  if $cert_source != '' {
+    file { '/etc/ssl/private' :
+      ensure => 'directory',
+      mode   => '0700',
+      owner  => 'root',
+      group  => 'root'
+    }
+
+    file { '/etc/ssl/private/www-lib':
+      ensure  => 'directory',
+      mode    => '0700',
+      owner   => 'haproxy',
+      group   => 'haproxy',
+      recurse => true,
+      purge   => true,
+      links   => 'follow',
+      notify  => Service['haproxy'],
+      source  => "puppet://${cert_source}/www-lib"
```

Comment:
Very weird that puppet-lint complains about this. Our Rakefile explicitly says to skip that check.

Reply:
Probably because I ran `puppet-lint` directly rather than using whatever was specified in the `Rakefile`.